bugbug/utils.py

```diff
@@ -276,18 +290,26 @@ def open_tar_zst(path: str, mode: str) -> Iterator[tarfile.TarFile]:
 # Using tar directly is twice as fast than through Python!
 def create_tar_zst(path: str) -> None:
     inner_path = path[: -len(".tar.zst")]
+    archive_name = f"{inner_path}.tar"
 
     if not os.path.exists(inner_path):
         raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), inner_path)
 
-    subprocess.run(["tar", "-I", "zstdmt", "-cf", path, inner_path], check=True)
+    subprocess.run(["tar", "-cf", archive_name, inner_path], check=True)
+    zstd_compress(archive_name)
+    os.remove(archive_name)
```

Comment:
This will slow things down, as we will have first to create a big tar file, and then compress it. It's better instead to compress in memory and directly save to disk the compressed file.

Reply:
See https://github.com/mozilla/bugbug/commit/90679941a7648dfac62bc402dc578581e473d6c9 and https://github.com/mozilla/bugbug/commit/b7fb7d7bca926bd711d848bfbd6476897e6b2553 for how to create/extract tar zst with Python only and without the zstd/zstdmt binaries.